docs/tutorials/new-feature-tutorial.md

```diff
@@ -64,6 +64,12 @@ organization in Zulip). The following files are involved in the process:
   admin settings pages.
 - `frontend_tests/node_tests/dispatch.js`
 
+**Documentation**
+
+- `zserver/openapi/zulip.yaml`: OpenAPI definitions for the Zulip REST API.
+- `templates/zserver/api/changelog.md`: documentation for changes to the Zulip Server API.
+- `templates/zserver/help/...`: user facing documentation for new features.
```

Comment:
`zerver` has a typo here.

Reply:
Not sure why I was imagining an 's' for all instances of `zerver` 😅